AppInspector/rules/default/frameworks/rust.json

```diff
@@ -0,0 +1,76 @@
+﻿[
+  {
+    "name": "Development Framework: Rocket",
+    "id": "AI025610",
+    "description": "Development Framework: Rocket",
+    "applies_to": [ "rust" ],
+    "tags": [ "Framework.Development.Rocket" ],
+    "severity": "moderate",
+    "patterns": [
+      {
+        "pattern": "extern crate rocket;|use rocket",
+        "type": "regex",
+        "scopes": [ "code" ]
+      }
+    ]
+  },
+  {
+    "name": "Development Framework: ActixWeb",
+    "id": "AI025620",
+    "description": "Development Framework: ActixWeb",
+    "applies_to": [ "rust" ],
+    "tags": [ "Framework.Development.ActixWeb" ],
+    "severity": "moderate",
+    "patterns": [
+      {
+        "pattern": "use actix_web",
+        "type": "regex",
+        "scopes": [ "code" ]
+    ]
```

Comment:
Missing ending brace

Reply:
Yeah that's the one found also.  I was just having issues with VS recognizing the file change to upload again but resolved now and will review remaining comments before pushing.